pkg/engine/loadbalancers.go

```diff
@@ -236,6 +237,93 @@ func CreateLoadBalancer(prop *api.Properties, isVMSS bool) LoadBalancerARM {
 	return loadBalancer
 }
 
+func createOutboundRules(prop *api.Properties) *[]network.OutboundRule {
+	currentVersion, _ := semver.Make(prop.OrchestratorProfile.OrchestratorVersion)
+	min13Version, _ := semver.Make("1.13.7")
```

Comment:
why do we need this? IMO we should always assume the earliest supported patch version of a k8s version is .0, this seems like it would require additional maintenance when more patch versions are added later and we deprecate these

Reply:
This is needed as `enableTcpReset` was cherry picked for v1.14.3 and v1.13.7.

Reply:
FYI - current k8s cloud provider has a bug that it won't set this same property to true in load balancer rules. I'm preparing a PR to fix that.